src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -1237,7 +1261,7 @@ class MNPlusOneDBSpanDetector(PerformanceDetector):
 
     def init(self):
         self.stored_problems = {}
-        self.state = SearchingForMNPlusOne(self.settings)
+        self.state = SearchingForMNPlusOne(self.settings, self.event())
```

Comment:
👀 `self.event()`? Is `event` also a callable?

Reply:
It is indeed! https://github.com/getsentry/sentry/blob/master/src/sentry/utils/performance_issues/base.py#L102-L103 Render blocking assets uses it too, e.g. https://github.com/getsentry/sentry/blob/master/src/sentry/utils/performance_issues/performance_detection.py#L441

Reply:
Oohh, I see, okay!